java/test/jmri/jmrix/can/adapters/gridconnect/sproggen5/serialdriver/CanisbConnectionConfigTest.java

```diff
@@ -12,17 +11,10 @@
  **/
 public class CanisbConnectionConfigTest extends jmri.jmrix.AbstractSerialConnectionConfigTestBase {
 
-    @Test
-    public void testCTor() {
-        CanisbConnectionConfig c = new CanisbConnectionConfig();
-        Assert.assertNotNull("exists",c);
-    }
```

Comment:
Why are these methods removed?

Reply:
Thanks for looking over these @danielb987 

There's already a CanisbConnectionConfig instance created in the setup method, cc.
This is checked for null in the super testCtor.

These testCTor methods were being flagged in Spotbugs as "VERY confusing to have methods " with the same name, only differences in capitalisation, especially when from a super class.